package.json

```diff
@@ -16,12 +16,11 @@
 		"@emotion/styled": "^10.0.27",
 		"@guardian/ab-core": "^5.0.0",
 		"@guardian/automat-modules": "^0.3.8",
-		"@guardian/commercial": "17.8.0",
-		"@guardian/consent-management-platform": "13.12.1",
+		"@guardian/commercial": "17.9.0",
```

Comment:
Could we stick to bumping one package only?

Reply:
Not in this case because commercial also requires the cmp and hence we also need the new version of commercial here that is pointing to libs for cmp.